osx/KA-Lite/KA-Lite.xcodeproj/project.pbxproj

```diff
@@ -79,13 +79,13 @@
 		F1DDD40D1A6E5DC10022F1C9 /* Products */ = {
 			isa = PBXGroup;
 			children = (
-				F1DDD40C1A6E5DC10022F1C9 /* KA-Lite-Monitor.app */,
-				F1DDD41F1A6E5DC10022F1C9 /* KA-Lite-MonitorTests.xctest */,
+				F1DDD40C1A6E5DC10022F1C9 /* KA-Lite.app */,
+				F1DDD41F1A6E5DC10022F1C9 /* KA-LiteTests.xctest */,
```

Comment:
We used `KA-Lite-Tests` above but you are using `KA-LiteTests` here @djallado - we should be consistent.